max_nn : 100
robot_radius : 0.2
x_max : 3.0
x_min : -1.0
y_max : 0.3
y_min : -0.3
loop_rate : 50
max_linear_vel : 1.0
max_angular_vel : 1.0
dt : 0.1
recovery_start_time : 0.3
dangerous_potential : false